src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Automation/Peers/AutomationPeer.cs

```diff
@@ -703,6 +703,20 @@ virtual protected int GetPositionInSetCore()
             return AutomationProperties.AutomationPositionInSetDefault;
         }
 
+        /// <summary>
```

Comment:
I think you should fill the xml doc, similar to what you did in IsDialogCore.

Reply:
Done